vita3k/modules/SceDriverUser/SceAppMgrUser.cpp

```diff
@@ -172,9 +200,17 @@ EXPORT(int, sceAppMgrForceUmount) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppMgrGameDataMount) {
-    TRACY_FUNC(sceAppMgrGameDataMount);
-    return UNIMPLEMENTED();
+EXPORT(int, sceAppMgrGameDataMount, const char *app_path, const char *patch_path, const char *rif_path, char *mount_point) {
+    TRACY_FUNC(sceAppMgrGameDataMount, app_path, patch_path, rif_path, mount_point);
+    STUBBED("Using strcpy");
+    if (strlen(app_path) > 0)
+        strcpy(mount_point, fmt::format("{}/", app_path).c_str());
```

Comment:
```suggestion
        fmt::format_to(mount_point, "{}/", app_path);
```
And same for all the other times this is used, this looks much cleaner.

Reply:
oh, i no know this